content/en/logs/workspaces/export.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Export Log Workspaces Datasets
+private: true
+further_reading:
+- link: "/logs/workspaces/"
+  tag: "Documentation"
+  text: "Learn more about Log Workspaces"
+- link: "/dashboards/widgets/list/"
+  tag: "Documentation"
+  text: "List Widget"
+---
+
+## Overview
+
+Export your Workspaces query to dashboards and notebooks. The export feature allows you to extract datasets and display them in dashboards and notebooks for analysis or reporting. 
+
+## Export a Workspaces dataset
```

Comment:
I think we need to align on some terminology here, in my head it's `workspace` -> `cell` -> `dataset` -> `dashboard/notebook`. 

We export a workspace `cell` as a `dataset` which can be used in notebooks/dashboards. 

Similarly, the use of `query` above this, would actually be a workspace `cell` instead.

Reply:
Agreed, thanks for catching.

This should then read:

```
Export your Workspaces cell as a dataset to dashboards and notebooks. The export feature allows you to create datasets and display them in dashboards and notebooks for analysis or reporting. 

## Export a Workspaces cell as a dataset 
```